### ISSUE TITLE
Set can values

### ISSUE THREAD

**Issue by danielxstefan** [hmq2018/Python-Vector-CANoe#2]
Hello,

How can i set a can signal from this library?
I don`t have any information regarding this topic.

**Comment by queen97king2019** [comment]
Hi,

Is there any option for using TestSetup using this Library?

**Comment by ceibay** [comment]
No, you cannot set a CAN signal value by using this library. You may write some CAPL codes to change CAN signal over a global variable, and control the variable with this lib

**Comment by ceibay** [comment]
> Hello,
> 
> How can i set a can signal from this library?
> I don`t have any information regarding this topic.

No, you cannot set a CAN signal value by using this library. You may write some CAPL codes to change CAN signal over a global variable, and control the variable with this lib

**Comment by ceibay** [comment]
> Hi,
> 
> Is there any option for using TestSetup using this Library?

What do you mean TestSetup? The test unit script in CAPL?

**Comment by queen97king2019** [comment]
> > Hi,
> > Is there any option for using TestSetup using this Library?
> 
> What do you mean TestSetup? The test unit script in CAPL?

TestSetup meaning, how to we add a .tse file into CANoe.

**Comment by ceibay** [comment]
I think you can add a global variable in the configuration of the tse, and control the global variable via this PyCANoe library.

**Comment by shlokk96** [comment]
> I think you can add a global variable in the configuration of the tse, and control the global variable via this PyCANoe library.

Hello, 
Can you show one example?

**Comment by shlokk96** [comment]
> I think you can add a global variable in the configuration of the tse, and control the global variable via this PyCANoe library.

Hello, 
How to start stop and pause testcase using this library?

**Comment by BugKillerXu** [comment]
> Hi,
> 
> Is there any option for using TestSetup using this Library?

hello，I have modified the library and add the function of set the value of CAN, but I have no access to upload the library that I have modified

**Comment by funnyrabbitvu** [comment]
> > Hi,
> > Is there any option for using TestSetup using this Library?
> 
> hello，I have modified the library and add the function of set the value of CAN, but I have no access to upload the library that I have modified

Hi man, could you please share it to me via email? thank you alot!
